tests/wpt/meta/content-security-policy/style-src/style-src-hash-blocked.html.ini

```diff
@@ -1,7 +1,4 @@
 [style-src-hash-blocked.html]
   expected: TIMEOUT
-  [Should not load style that does not match hash]
-    expected: FAIL
-
   [Should fire a securitypolicyviolation event]
```

Comment:
We fail all of these as the events are fired on `window` instead of `document`. If you change it to `window.addEventListener` then it works.

Strangely enough, I thought that we did this, where we use the `document.global()` to fire the event: https://github.com/servo/servo/blob/2fa96cf961140cf0c0b31163844c71cb2bdf93c5/components/script/dom/document.rs#L4324

Reply:
We use the provided global as the target of the event: https://github.com/servo/servo/blob/2fa96cf961140cf0c0b31163844c71cb2bdf93c5/components/script/security_manager.rs#L167